scope.py

```diff
@@ -603,6 +605,31 @@ def train(
         if pre_trained_model is not None:
             classifier.load(pre_trained_model)
 
+        time_tag = datetime.datetime.utcnow().strftime("%Y%m%d_%H%M%S")
+
+        if not kwargs.get("test", False):
+            wandb.login(key=self.config["wandb"]["token"])
```

Comment:
A link to where to documentation for where to grab one's token?

Reply:
W&B's wizard will actually print out the url if you're running it the first time from the command line. Added anyways.